evergreen/evergreen.yml

```diff
@@ -510,9 +518,8 @@ functions:
     - command: shell.exec
       params:
         script: |
-          cd ${DRIVERS_TOOLS}/.evergreen/ocsp
-          /cygdrive/c/python38/python.exe -m venv ./venv
```

Comment:
was it a bug here in the path (and below)?

compare:
> /cygdrive/c/python38/python.exe

and

>/cygdrive/c/python/python38/python.exe

Reply:
They all use `/cygdrive/c/python/python38/python.exe` now from prepare-ocsp.sh

Reply:
yep, I just wanted to highlight that we probably had a bug here. So, not sure why it didn't affect our tests.. cc: @rstam , @JamesKovacs